tests/sqllogictests/suites/query/02_function/02_0060_function_geo

```diff
@@ -51,6 +51,9 @@ select point_in_polygon((3., 3.), [(6, 0), (8, 4), (5, 8), (0, 2)])
 ----
 1
 
+statement error 1065
+select point_in_polygon((3,), [(6, 0), (8, 4)])
```

Comment:
The `point_in_polygon` function has three types of arguments, but only one type has a test case, we need to add more tests to cover all the types.